s/icons/iconify/no-data.svg.ts

```diff
@@ -0,0 +1,81 @@
+import { svg } from "lit";
+
+export default svg`
+	<svg width="132" height="111" viewBox="0 0 132 111" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this svg file seems a lot bigger than it should be. perhaps optimize with `svgomg` online tool

Reply:
Fixed